src/context/videoContext.js

```diff
@@ -1,6 +1,7 @@
 import { useFetch } from "hooks";
+import { filterData } from "utils/videoUtils";
 
-const { createContext, useContext } = require("react");
+const { createContext, useContext, useState } = require("react");
```

Comment:
This is new to me. Is there a specific reason as to why the imports are done like this ?

Reply:
Thanks for pointing it out. I have been doing auto imports maybe my ide has auto imported it like we do in Node JS. will change this now..